packages/react/src/components/collapsible/tests/CollapsibleInfoBox.spec.tsx

```diff
@@ -41,13 +40,13 @@ describe('CollapsibleInfoBox', () => {
 
         it('should render a <h6 /> and <svg /> in header', () => {
             expect(component.find('h6').exists()).toBe(true);
-            expect(component.find(Svg).length).toBe(1);
+            expect(component.find('svg').length).toBe(2);
         });
 
         it('should not render a <svg /> in header if isSection', () => {
             mountComponent({isSection: true});
             expect(component.find('h6').exists()).toBe(true);
-            expect(component.find(Svg).length).toBe(0);
+            expect(component.find('svg').length).toBe(1);
         });

```

Comment:
The test is a bit weird `"should not render a <svg />"` ...  `expect(component.find('svg').length).toBe(1);`

Would it be easy to switch to RTL?

Reply:
Yea probably, one svg is the chevron indicating the collapsed state, which has nothing to do with the info icon. I didn't want to redo the tests here because the component is very ugly and probably shouldn't be used.